.github/workflows/ci.snyk.yml

```diff
@@ -0,0 +1,18 @@
+name: Snyk
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check vulnerabilities
+        uses: snyk/actions/node@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
```suggestion
          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
        with:
          args: --yarn-workspaces --strict-out-of-sync=false --detection-depth=5
```

- If we don't specify `--yarn-workspaces`, it will default to only uploading the `package.json` or treating it as a normal Yarn project
- `--strict-out-of-sync` is needed because of a bug with Yarn workspaces https://github.com/snyk/snyk/issues/350#issuecomment-762956088
- `--detection-depth=5` is needed in order to pick up `packages/backend/bindings/node`